### ISSUE TITLE
Errors while building wheel on Windows

### ISSUE THREAD

**Issue by alexandrul** [closeio/ciso8601#76]
```
module.c(508): error C2440: 'function': cannot convert from 'double' to 'const char *'
module.c(508): warning C4024: 'PyModule_AddStringConstant': different types for formal and actual parameter 3
module.c(508): error C2143: syntax error: missing ')' before 'constant'
module.c(508): error C2059: syntax error: ')'
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional\\VC\\Tools\\MSVC\\14.15.26726\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
Failed building wheel for ciso8601
```

Python 3.7.0 x64
VS Professional 2017 (15.8.5)
Windows 7 SP1 x64

**Comment by gillesdouaire** [comment]
Also Happening for Python 2.7, Under Windows 10. Happens with the latest release V2.1.0

**Comment by alexandrul** [comment]
I can't explain why, but it compiles just fine with one of the below variants:
```
define_macros=[("CISO8601_VERSION", '""""%s""""' % VERSION)]
define_macros=[("CISO8601_VERSION", '"""%s"""' % VERSION)]
```

**Comment by thomasst** [comment]
@movermeyer Can you take a look?

**Comment by movermeyer** [comment]
On it.

**Comment by movermeyer** [comment]
Alright, I've [fixed it](#77). I used the ["stringizing operator"](https://docs.microsoft.com/en-us/cpp/preprocessor/stringizing-operator-hash) as seen in [the example](https://github.com/fancycode/pylzma/commit/080ad8f50d9912b6f8b55b1fe109c83b5bdece79) provided by @alexandrul, as it provides a nice cross-platform solution.

New release should happen shortly. Sorry about this.

**Comment by CheckYourFax** [comment]
C:\Program Files (x86)\Windows Kits\10\include\10.0.26100.0\ucrt\wchar.h(313): warning C4013: '_mm_loadu_si64' undefined; assuming extern returning int
      C:\Program Files (x86)\Windows Kits\10\include\10.0.26100.0\ucrt\wchar.h(313): error C2440: 'initializing': cannot convert from 'int' to '__m128i'
      isocalendar.c(291): warning C4028: formal parameter 2 different from declaration
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit code 2
